### ISSUE TITLE
[Bug]: Queueing STX results in error 

### ISSUE THREAD

**Issue by sleepytanya** [MetaMask/metamask-mobile#10269]
### Describe the bug

When a user queues smart transactions and the first transaction gets confirmed, the app displays an error.

@forest-diggs-consensys We don't support the queueing of smart transactions on the Extension, is this functionality expected to be disabled on mobile?

### Expected behavior

-

### Screenshots/Recordings

https://github.com/MetaMask/metamask-mobile/assets/104780023/99398d1e-1929-4ce9-94f7-bac058e89429

<img width="438" alt="Screenshot 2024-07-05 at 15 59 03" src="https://github.com/MetaMask/metamask-mobile/assets/104780023/b1a8dec6-2793-4a33-a24a-f985812e4924">

### Steps to reproduce

1. Send STX
2. Send another STX while the first transaction is still pending
3. See the error after the first STX is confirmed

### Version

7.26.0

### Build type

None

### Device

Samsung S24+

### Operating system

Android

**Comment by forest-diggs-consensys** [comment]
Yes, this should be disabled on Mobile as well - @dan437 or @dbrans can you take a look at this? I'm seeing some funkiness with the activity list in @sleepytanya 's screen recording as well.

**Comment by sleepytanya** [comment]
@forest-diggs-consensys The funkiness been there for a long time (it's not related to the current release) - pending and cancelled transactions may disappear for a short time. 

https://github.com/MetaMask/metamask-mobile/issues/8762

**Comment by sleepytanya** [comment]
Present in v7.29.0, sometimes the app crashes after queueing STX:

![IMG_0113](https://github.com/user-attachments/assets/444a1d2c-4918-443c-949a-1b4d52a60611)

**Comment by forest-diggs-consensys** [comment]
I'm having trouble reproducing this now

**Comment by sleepytanya** [comment]
@forest-diggs-consensys 
 
I was unable to replicate the issue on version 7.35.0 (1469). In the latest RC, it appears users have the capability to queue STX transactions, though I'm unsure if this behavior is intended. Additionally, there seems to be a delay in the appearance of the 'Submitting your transaction' notification (weird transaction history behavior is a pre-existing bug):




https://github.com/user-attachments/assets/1d417633-37a5-430c-89b0-aa8f99653862

**Comment by sleepytanya** [comment]
Closing as the issue is resolved in the RC 7.37.0 as well - 2 consecutive STX can be be successfully sent:

https://github.com/user-attachments/assets/f7080498-0398-4702-95b4-1157e677765e
